lib/Elastica/Node/Info.php

```diff
@@ -208,16 +208,14 @@ public function refresh(array $params = [])
     {
         $this->_params = $params;
 
-        $path = '_nodes/'.$this->getNode()->getId();
+        $endpoint = new \Elasticsearch\Endpoints\Cluster\Nodes\Info();
+        $endpoint->setNodeID($this->getNode()->getId());
 
         if (!empty($params)) {
-            $path .= '/';
-            foreach ($params as $param) {
-                $path .= $param.',';
-            }
+            $endpoint->setMetric(join(',', $params));
```

Comment:
Wow, I just checked official client code, "join" even not needed, we call setMetric with array, https://github.com/elastic/elasticsearch-php/blob/master/src/Elasticsearch/Endpoints/Cluster/Nodes/Info.php#L31 . Will change it at evening and looks like we haven't test for this case.